src/Symfony/Component/Profiler/Profiler.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Profiler;
+
+use Symfony\Component\HttpKernel\Profiler\GenericProfileData;
```

Comment:
This GenericProfileData is here only for BC ? Too bad to create an hard dependency on HttpKernel component for a deprecated class :/